tests/units/test_openstackmap.py

```diff
@@ -0,0 +1,291 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FIWARE project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+
+__author__ = 'gjp'
+
+from os import environ
+from unittest import TestCase
+from skuld.openstackmap import OpenStackMap
+from mock import patch, MagicMock
+from collections import defaultdict
+import os
+from requests import Response
+
+
+class MySessionMock(MagicMock):
+    # Mock of a keystone Session
+
+    def get_endpoint(self, auth, **kwargs):
+        
+        endpoint = "http://cloud.host.fi-ware.org:4731/v2.0"
+        return endpoint
+
+    def get_access(self, session):
+
+        service2 = {u'endpoints': [{u'url': u'http://83.26.10.2:4730/v3/',
+                                    u'interface': u'public', u'region': u'Spain2',
+                                    u'id': u'00000000000000000000000000000002'},
+                                   {u'url': u'http://172.0.0.1:4731/v3/', u'interface': u'administator',
+                                    u'region': u'Spain2', u'id': u'00000000000000000000000000000001'}],
+                                    u'type': u'identity', u'id': u'00000000000000000000000000000045'}
+
+        d = defaultdict(list)
+        d['catalog'].append(service2)
+
+        return d
+
+    def request(self, url, method, **kwargs):
+
+        resp = Response()
+
+        if url == '/servers/detail?all_tenants=1':
+            text = '{"servers": [{"status": "ACTIVE", "updated": "2015-11-23T08:08:58Z", ' \
```

Comment:
from my point of view, I usually take out in a file all the text content in order to clarify the text and put these files inside the resource directory. @jframos what do you think?